packages/storybook/stories/additional-docs.js

```diff
@@ -156,6 +154,12 @@ export const additionalDocs = {
     maturityCategory: USE,
     maturityLevel: DEPLOYED,
   },
+  'va-button': {
+    guidanceHref: 'button',
+    guidanceName: 'Button',
+    maturityCategory: USE,
+    maturityLevel: BEST_PRACTICE,
```

Comment:
Can we please set this to "use with caution: candidate" until it goes through staging review?